workloads/kube-burner/README.md

```diff
@@ -193,17 +192,16 @@ Each iteration creates the following objects:
 
 ### Churn
 
-Churning a workload allows you to scale down and then up a percentage of pods or namespaces **AFTER** the workload has run, but before cleanup (if enabled). It takes in a percentage, **CHURN_PERCENT**, which is the percentage of JOB_ITERATIONS it churns during each churn cycle. If the **pod** CHURN_TYPE is used then it will delete and recreate all the pods in a namespace. If the **namespace** CHURN_TYPE is used then it will delete and recreate **ALL** the resources in the namespace. After each churn cycle it will sleep for the remainder of **CHURN_WAIT** seconds if the time it took to churn was less than the **CHURN_WAIT** otherwise it continues immediately. This loop continues until the **CHURN_DURATION** has elapsed.
+Churning a workload allows you to scale down and then up a percentage of **JOB_ITERATIONS** after the objects have been created. It takes in a percentage, **CHURN_PERCENT**, which is the percentage of JOB_ITERATIONS it churns during each cycle. It will delete and recreate **ALL** the objects specified in the job definition. After each churn cycle it will sleep for the **CHURN_DELAT**. This loop continues until the **CHURN_DURATION** has elapsed.
```

Comment:
```suggestion
Churning a workload allows you to scale down and then up a percentage of **JOB_ITERATIONS** after the objects have been created. It takes in a percentage, **CHURN_PERCENT**, which is the percentage of JOB_ITERATIONS it churns during each cycle. It will delete and recreate **ALL** the objects specified in the job definition. After each churn cycle it will sleep for the **CHURN_DELAY**. This loop continues until the **CHURN_DURATION** has elapsed.
```